CantoboardFramework/Keyboard/View/Keyboard/PhoneKeyboardViewLayout.swift

```diff
@@ -20,30 +20,30 @@ class PhoneKeyboardViewLayout : KeyboardViewLayout {
     
     static let numbersHalf: [[[KeyCap]]] = [
         [["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]],
-        [["-", "/", ":", ";", "(", ")", .currency, .doubleQuote, "「", "」"]],
-        [[.keyboardType(.symbolic)], [".", ",", "、", "&", "?", "!", .singleQuote], [.backspace]],
+        [["-", "/", ":", ";", "(", ")", .currency, .doubleQuote, "｢", "｣"]],
+        [[.keyboardType(.symbolic)], [".", ",", "､", "^_^", "?", "!", .singleQuote], [.backspace]],
         [[.keyboardType(.alphabetic(.lowercased)), .nextKeyboard], [.space(.space)], ["@", .returnKey(.default)]]
     ]
     
     static let symbolsHalf: [[[KeyCap]]] = [
         [["[", "]", "{", "}", "#", "%", "^", "*", "+", "="]],
-        [["_", "—", "\\", "|", "~", "<", ">", "«", "»", "•"]],
-        [[.keyboardType(.numeric)], [".", ",", "、", "^_^", "?", "!", .singleQuote], [.backspace]],
-        [[.keyboardType(.alphabetic(.lowercased)), .nextKeyboard], [.space(.space)], [.returnKey(.default)]]
+        [["_", "\\", "|", "~", "<", ">", "«", "»", "&", "·"]],
```

Comment:
The default iOS keyboard uses "•" over "·". Why are we picking "·"?

Reply:
I can revert it. I did think that the interpunct is more important than the bullet dot though.

Reply:
Another reason is that I don’t want `•` `·` to be a halfwidth-fullwidth relationship.

Reply:
Another reason is that I don’t want `•` ↔︎ `·` to be a halfwidth-fullwidth relationship.